README.md

```diff
@@ -1,85 +1,136 @@
 
-**Scenario 2: Applied AI for transformations on content**
+## Prerequisites
 
-Indexers drive the native AI enrichment capabilities of Cognitive Search through skillsets. If your scenario requires AI enrichment but you can't use indexers, you can replace a skillset by making direct calls to Cognitive Services. To illustrate this technique, the demo invokes the following resources:
++ [Visual Studio Code](https://code.visualstudio.com/download), with a [C# extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp) and [.NET Core tools](https://marketplace.visualstudio.com/items?itemName=formulahendry.dotnet)
++ [Azure Cognitive Search](https://docs.microsoft.com/azure/search/search-create-service-portal), Basic or above
++ [Azure Cognitive Services](https://docs.microsoft.com//azure/cognitive-services/cognitive-services-apis-create-account), multi-service and multi-region
++ [Azure Cosmos DB (SQL API)](https://docs.microsoft.com/azure/cosmos-db/sql/how-to-create-account)
++ [Azure Storage](https://docs.microsoft.com/azure/storage/common/storage-account-create)
++ An [Azure Function app](https://docs.microsoft.com/azure/azure-functions/functions-create-function-app-portal#create-a-function-app), with a runtime stack of .NET 6 on a Windows operating system. The following screenshot illustrates the configuration.
 
-+ [Azure Cognitive Service for Language](https://docs.microsoft.com/azure/cognitive-services/language-service/overview) for language detection, entity recognition, and sentiment analysis.
+   ![Screenshot of the create function app.](readme-images/create-function-app.png)
 
-+ [OCR Read API](https://docs.microsoft.com/azure/cognitive-services/computer-vision/overview-ocr) in [Computer Vision](https://docs.microsoft.com/azure/cognitive-services/computer-vision/overview) to recognize and read strings from PDFs.
+## Set up storage and get connection information
 
-This sample stores the finished output in a Cosmos DB database. 
+Use Azure portal to create databases and containers for storing data in Cosmos DB and Azure Storage. While in the portal, get the keys and connection strings you'll need for establishing connections. You'll paste these values into the local.settings.json file in a later step.
 
-Indexing operates over the stored data. The sample uses the [**Azure.Search.Documents**](https://www.nuget.org/packages/Azure.Search.Documents/) library from the Azure SDK for .NET to create, load, and query the search index.
+1. In Cosmos DB, use **Data Explorer** to create a database named "Wikipedia" and a container named "pages". The "pages" container will eventually store Wikipedia pages as JSON documents.
 
-## Objects created in this demo
+1. Create a second database named "Wikipedia-Knowledge-Store" and a container named "enriched-output". This database receives all output generated by Cognitive Services, whether its from JSON file updates in the "pages" container, or PDF file updates in Azure Storage.
```

Comment:
I'm not sure if this matters or not, but I put the Knowledge Store container in the same database.  There is no reason it can't be in a different database, we'll need to test to make sure this doesn't require any code/connection string changes.

Reply:
I used a separate database.  The KS connection info, including db name is in local.app.settings. The Cosmos source database name and container is specified in the function.  I've run this configuration a lot.